server/cmwell-cons/src/main/scala/DataInitializer.scala

```diff
@@ -134,7 +134,8 @@ class DataInitializer(h: Host, jwt: String, rootDigest: String, rootDigest2: Str
       "root" -> s"""{"digest":"$rootDigest","digest2":"$rootDigest2","paths":[${allowRoot("rw")}],"operations":["Admin","Overwrite","PriorityWrite"],"roles":[]}""",
       "pUser" -> s"""{"paths":[${allowRoot("rw")}],"operations":["Admin","Overwrite","PriorityWrite"],"roles":[]}""",
       "dca" -> s"""{"paths":[${allowRoot("rw")}],"operations":["Overwrite"],"roles":[]}""",
-      "docu" -> """{"paths":[{"id":"/","recursive":true,"sign":"+","permissions":"r"},{"id":"/meta/docs","recursive":true,"sign":"+","permissions":"rw"}],"rev":0}"""
+      "docu" -> """{"paths":[{"id":"/","recursive":true,"sign":"+","permissions":"r"},{"id":"/meta/docs","recursive":true,"sign":"+","permissions":"rw"}],"rev":0}""",
+      "stp3" -> s"""{"paths":[${allowRoot("rw")}],"operations":["Overwrite"],"roles":[]}"""
```

Comment:
1. stpAgent is more suitable name. BTW the token is configured to "stp" user and won't work
1. overwrite is not needed for stp
1. the stp agent shouldn't have so strong permissions (e.g. write to everywhere). it should be able to write to any data but no more. /meta should be restricted. Nevertheless, it writes the persistancy tokens and only that specific meta location should be enabled.